### ISSUE TITLE
match quest numbering in URLs and web page

### ISSUE THREAD

**Issue by JTFouquier** [JTFouquier/mark2cure#27]
Not a huge deal, but the mismatch between the quest number in the URL and the number shown in the web page is weird.  Would be good to get those to match.

![12-2-2014 4-28-41 PM.jpg](https://bitbucket.org/repo/y7oq9p/images/3460039569-12-2-2014%204-28-41%20PM.jpg)

----------------------------------------
- Bitbucket: https://bitbucket.org/sulab/mark2cure/issue/27
- Originally reported by: [Andrew Su](http://bitbucket.org/asu)
- Originally created at: 2014-12-03T01:31:18.219

**Comment by JTFouquier** [comment]
Yeah.. I'm not sure about this.  I agree its weird.  However the URL-based interaction form the user side is not really there yet anyway.  e.g. if I start plugging in numbers into the page id slot manually the behavior is clearly unintentional.  I was doing this and hit a 500 error at one point but am struggling to reproduce it.  Now, I can use the URLs to get places I can't through the clicking interface - like back to completed quests. So if I use the url to get back to completed quest, it loads a document as before, but then gives me an alert on submit that there was a problem submitting the request.  Nice that it doesn't just 500 out, but also not great experience.  

If possible, it might be better to simple hide the URLs in this case such that users would be forced to only use the interface to jump around.  Would probably produce less headaches.  The alternative is to implement all the appropriate logic such that all of the potentially accessible views made sense to the user or at least routed them back to a logical place like the dashboard.



----------------------------------------
Original comment by: [bgood](http://bitbucket.org/bgood)

**Comment by x0xMaximus** [comment]
@gtsueng  I don't expect this to still be an issue as I recall rebuilding the quest naming index a long time ago. Is this safe to close or have there been more recent discrepancies with the url scheme and the titles that we present in the header section of the pages?

**Comment by x0xMaximus** [comment]
Some concepts to outline:

Task - the Quest Identification number you see in the URL (task_task.id)
Quest Name - the Quest Index in the group (task_task.name)

While this issue addresses a naming inconsistancy, it actually address a larger problem in the DB scheme given the addition of 2 task types: ER Tasks (task_task) should live inside the tasks.entity_recognition application
